services/task-svc/migrations/000005_init.down.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE tasks
```

Comment:
@DasProffi I've updated the down migration. We should `ALTER` the `tasks` table back instead of `DROP`ing.

Reply:
yes, thank you ^^